.eslintrc.js

```diff
@@ -1245,7 +1272,7 @@ module.exports = {
           files: [
             'x-pack/packages/security-solution/features/**/*.{js,mjs,ts,tsx}',
             'x-pack/packages/security-solution/navigation/**/*.{js,mjs,ts,tsx}',
-            'x-pack/plugins/security_solution/**/*.{js,mjs,ts,tsx}',
+            'x-pack/plugins/integration_assistant/**/*.{js,mjs,ts,tsx}',
```

Comment:
I did some digging into the issue and it looks like it's this line that causes the `type` to be removed from imports. 

Reverting this line should fix the issue.